src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -18,22 +18,25 @@ public class CommandResult {
 
     /** The application should exit. */
     private final boolean exit;
+    private final Class<?> executedCommand;
+
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, Class<?> executedCommand) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
         this.exit = exit;
+        this.executedCommand = executedCommand;
     }
```

Comment:
Sorry but could you explain the reason for including an argument for executedCommand in the constructor? I may have missed it but I'm unable to find it's usage

Reply:
Removed! Was extra for implementing ui